tests/tests.py

```diff
@@ -254,3 +257,36 @@ def handler(sender, username, **kwargs):
 
         self.assertTrue(self.signal_was_called)
 
+    def test_exceed_token_amount_per_user(self):
+
+        with override_settings(REST_KNOX=token_user_limit_knox):
+            reload_module(views)
+            for _ in range(10):
+                token = AuthToken.objects.create(user=self.user)
+            url = reverse('knox_login')
+            self.client.credentials(
+                HTTP_AUTHORIZATION=get_basic_auth_header(self.username, self.password)
+            )
+            response = self.client.post(url, {}, format='json')
+            reload_module(views)
```

Comment:
this should be indented outside `override_settings` block

Reply:
I have fixed this in latest commit.